TechForPR/chefsForPR
# New Request form in Spanish

# Solves issue Number: #4 

## Description

It creates a form that lets visitors submit a new meal request

## Todos

Abstract the code further, is a bit of a mess, on actions.js, the model and routes/index.

## Deploy Notes

Added a new package called [forms](https://www.npmjs.com/package/forms) To 
generate the HTML for the forms.

## Steps to Test or Reproduce

Visit `http://localhost:3000/request/new` and try creating a new Request, if all the data is complete it will succeed and show the confirmation number, if it fails it should say so and display error on the missing fields.

## Impacted Areas in Application

It creates a new route `http://localhost:3000/request/new` and allows to create food requests.
